src/ontology/mondo-ingest.Makefile

```diff
@@ -426,21 +426,24 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
+# todo: is `$(MAKE) pip-bioregistry` still necessary? What version of bioregistry was needed at the time this line was added?
 $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db $(MAPPINGSDIR)/rejected-mappings.tsv
-	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
-	rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical-2.sssom.tsv
+	@rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv
+	@rm -f $(MAPPINGSDIR)/mondo-sources-all-lexical-2.sssom.tsv
 	$(MAKE) pip-bioregistry
```

Comment:
## `$(MAKE) pip-bioregistry` still needed?
Added this todo:
> `# todo: is `$(MAKE) pip-bioregistry` still necessary? What version of bioregistry was needed at the time this line was added?`

@matentzn Do you think this is still necessary? Git blame shows that you added it Jan 31st of this year. Maybe ODK has been sufficiently updated?

Reply:
Probably not!

Reply:
Alright, gonna remove